R/fun_makefile_create.R

```diff
@@ -36,52 +36,69 @@ deepstate_create_makefile <-function(package,fun_name){
   rinside_lib <- system.file("lib", package="RInside")
 
   # CPPFLAGS : headers inclusion
-  compiler_cppflags <- paste0("-I", rcpp_include) 
-  compiler_cppflags <- paste(compiler_cppflags, paste0("-I", rcppdeepstate_include)) 
-  compiler_cppflags <- paste(compiler_cppflags, paste0("-I", rcpparmadillo_include)) 
-  compiler_cppflags <- paste(compiler_cppflags, paste0("-I", rinside_include)) 
-  compiler_cppflags <- paste(compiler_cppflags, paste0("-I", deepstate_build))
-  compiler_cppflags <- paste(compiler_cppflags, paste0("-I", deepstate_include)) 
-  compiler_cppflags <- paste(compiler_cppflags, paste0("-I", qs_include)) 
-  compiler_cppflags <- paste(compiler_cppflags, paste0("-I", "${R_INCLUDE}"))
-  write_to_file <- paste0(write_to_file, "CPPFLAGS=",compiler_cppflags, "\n")
+  cppflags <- c(rcpp_include, rcppdeepstate_include, rcpparmadillo_include,
+                rinside_include, deepstate_build, deepstate_include, qs_include,
+                "${R_INCLUDE}")
+
+  compiler_cppflags <- paste(cppflags, collapse=" -I ")
+  cppflags_write <- paste0("CPPFLAGS=-I",compiler_cppflags)
```

Comment:
Is this logic tested? I thought there was supposed to be no between -I and the include path? 
anyway please remove repetition of -I in this logic. I believe it should be `paste(paste0("-I", cppflags), collapse=" ")`

Reply:
Good catch! There should be no space between `-I` and the include path. I tested this locally and it works, but I'll let CI finish to ensure that everything is working correctly. 

Updated with commit a6f1375